### PR TITLE
Fix the help message for stageout script

### PR DESCRIPTION
**Pull request by connor-mcisaac**
Just a quick change to the help message of pycbc_stageout_failed_workflow

### CODE REVIEW

File: bin/pycbc_stageout_failed_workflow [hunk 1]
```diff
@@ -14,7 +14,7 @@ while true
 do
     case "$1" in
 	-h|--help)
-	    echo "usage: pycbc_copy_output_map [-h] input_map output_map [optional arguments]"
+	    echo "usage: pycbc_copy_output_map [-h] workflow_directory [optional arguments]"
```
Comment (spxiwh) [inline bin/pycbc_stageout_failed_workflow:17]: Shouldn't that be -d workflow_directory?

**Review by spxiwh — APPROVED** [review]
I'll give approval as it's a simple fix, but see comment .... Also tests seem to have an issue right now, which would need resolving first, but that is unrelated to this PR.

**Comment by connor-mcisaac** [comment]
No problem I have updated the help messages for the stageout and copy map scripts.

**Comment by spxiwh** [comment]
Thanks @connor-mcisaac this is approved for merge, but it will need a rebase to pick up the fix for the failing test suite.

**Pull request closed without merging**